### ISSUE TITLE
error build the package

### ISSUE THREAD

**Issue by lidaof** [molstar/molstar#75]
Hi there,

I am trying to run npm run build and looking for examples...get following error, can anyone suggest how to fix this? Thanks a lot!

```
$ npm run build-viewer

> molstar@1.1.4 build-viewer /Users/d/web/molstar
> npm run build-tsc && npm run build-extra && npm run build-webpack-viewer


> molstar@1.1.4 build-tsc /Users/d/web/molstar
> concurrently "tsc --incremental" "tsc --build tsconfig.commonjs.json --incremental"

[0] tsc --incremental exited with code 0
[1] src/servers/volume/server/web-api.ts(156,43): error TS2532: Object is possibly 'undefined'.
[1] tsc --build tsconfig.commonjs.json --incremental exited with code 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! molstar@1.1.4 build-tsc: `concurrently "tsc --incremental" "tsc --build tsconfig.commonjs.json --incremental"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the molstar@1.1.4 build-tsc script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/d/.npm/_logs/2020-07-26T04_52_58_250Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! molstar@1.1.4 build-viewer: `npm run build-tsc && npm run build-extra && npm run build-webpack-viewer`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the molstar@1.1.4 build-viewer script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/d/.npm/_logs/2020-07-26T04_52_58_357Z-debug.log
```

**Comment by dsehnal** [comment]
I suspect the issue was some breaking change in the latest version of the TypeScript compiler as I haven't seen this error a couple weeks ago. Should be fixed in bea462b.

**Comment by lidaof** [comment]
interesting....doesn't realize such a small change can fix this. thank you!
